docs/_data/australia-2022-config.yaml

```diff
@@ -0,0 +1,182 @@
+name: Australia
+
+shortcode: australia # australia australia?
+year: 2022
+city: Melbourne
+local_area: Australia
+area: Australia
+area_adj: Australian
+tz: AEDT
+email: australia@writethedocs.org
+color: green
+rgb: 61, 217, 74
+hex: 3dd94a
+time_format: 12h
+
+photos:
+  default: _static/conf/images/headers/WTDHeader.png
+
+buttons:
+  top:
+  #   - text: Welcome to our conference
+  #     link: /news/welcome
+    # - text: Sponsor us
+    #   link: /sponsors/prospectus
+    # - text: Submit a Talk
+    #   link: /cfp
+    # - text: Date and ticket information
+    #   link: /news/announcing-virtual-conf-dates-tickets
+    # - text: Read our conference guide
+    #   link: /welcome-wagon
+    # - text: See the Schedule!
+    #   link: /schedule
+    # - text: Buy a Ticket!
+    #   link: /tickets
+    # - text: See the talks!
+    #   link: /speakers
+ #    - text: See the Schedule!
+ #      link: /schedule
+ #    - text: Read our guide
+ #      link: /welcome-wagon
+    # - text: Watch the Live Stream
+    #   link: /livestream
+    # - text: Browse the photos
+    #   link_absolute: https://www.flickr.com/photos/writethedocs/albums/72157691507514803
+    # - text: Watch the videos!
+    #   link_absolute: https://www.youtube.com/playlist?list=PLZAeFn6dfHpkBJAPYFrob6gqdiBuePwGJ
+  bottom:
+  #   - text: Welcome to our conference
+  #     link: /news/welcome
+    # - text: Sponsor the conference
+    #   link: /sponsors/prospectus
+    # - text: Submit a Talk
+    #   link: /cfp
+    # - text: See the Schedule!
+    #   link: /schedule
+    # - text: Date and ticket information
+    #   link: /news/announcing-virtual-conf-dates-tickets
+    # - text: See the talks!
+    #   link: /speakers
+    # - text: Buy a Ticket!
+    #   link: /tickets
+ #    - text: Read our guide
+ #      link: /welcome-wagon
+ #    - text: See the schedule!
+ #      link: /schedule
+    # - text: Watch the Live Stream
+    #   link: /livestream
+    # - text: Browse the photos
+    #   link_absolute: https://www.flickr.com/photos/writethedocs/albums/72157691507514803
+    # - text: Watch the videos!
+    #   link_absolute: https://www.youtube.com/playlist?list=PLZAeFn6dfHpkBJAPYFrob6gqdiBuePwGJ
+
+tickets:
+  corporate:
+    price: $200
+  independent:
+    price: $125
+  student:
+    price: $25
+
+sponsorship:
+  first_draft:
+    price: AUD $1,000
+  second_draft:
+    price: AUD $2,000
+  publisher:
+    price: AUD $4,000
+  patron:
+    price: AUD $7,000
+  keystone:
+    price: AUD $10,000
+
+date: # how do we handle these? human readable would be nice.
+  main: "**December 8-9, 2022**"
+  short: Dec 8-9, 2022
+  tickets_live: "July 2022"
+  month: May
+
+  # Conference is special for the index to show the combined conference days
+  conference:
+    event: Main Conference
```

Comment:
Do we want to experiment with a shorter day 2 and half-day writing day in the afternoon?